webpack/components/OrganizationCheck/__tests__/OrganizationCheck.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import OrganizationCheck from '../../OrganizationCheck';
+
+describe('Organization check page', () => {
+  /* eslint-disable react/jsx-closing-tag-location */
```

Comment:
I've gotten around this before by setting the element to a `const`.  We may want to revisit this rule in the future.